packages/babel-plugin-short-css-vars/test/fixtures/short-css-vars/ignore-string/options.json

```
@@ -0,0 +1,7 @@
+{
+  "plugins": [
+    ["../../../../", {
+      "ignore": "two"
```

maybe put something in the string to make it obvious that it's converted to regex?  maybe something like 
```suggestion
      "ignore": "^.+-two"
```